backend/routes/shops.js

```diff
@@ -873,6 +877,23 @@ module.exports = function (router) {
         additionalOpts.printfulWebhookSecret = ''
       }
 
+      if (req.body.paypal) {
+        log.info(`Shop ${shopId} - Registering PayPal webhook`)
+        await paypalUtils.deregisterWebhook(shopId, existingConfig)
```

Comment:
Can we check the webhook doesn't already exist instead of deleting / recreating every time we PUT /shop/config?